lms/envs/common.py

```diff
@@ -3564,3 +3564,6 @@ def _make_locale_paths(settings):
 # CDN LINK
 
 CDN_LINK = "https://static.philanthropyu.org/"
+
+# PHILU Configuration Parameters.
+PHILU_DEFAULT_TERM_SEARCH_LENGTH = 2
```

Comment:
is this specifically related to organization? If so how about we rename this constant with something like `PHILU_ORGANIZATION_SEARCH_LENGTH `. Also this is related to on boarding don't you think this constant should be inside onboarding app. Thoughts?

Reply:
@monofal we can use it anywhere we want, as there may be other places where we want to use term length.